addon.py

```
@@ -1335,118 +1506,259 @@ def import_generated_asset_fal_ai(self, request_id: str, name: str):
+    bl_label = "Ask LLM About Scene"
+
+    def execute(self, context):
+        from .llm_handler import query_llm
+
+        image_path = render_and_save_image()
+        metadata = extract_scene_summary()
+        backend = context.scene.mcp_llm_backend
+
+        if backend == "ollama":
+            ollama_model_name = context.scene.mcp_ollama_model_name
+            response = query_llm(backend=backend, image_path=image_path, metadata=metadata, ollama_model_name=ollama_model_name)
```

Pylint (3.3.7)

[convention] Line too long (128/100)

(C0301)